### PR TITLE
Send player message when clicking on advancements or green book.

### PR DESCRIPTION
**Pull request by Seadragon91**
Instead of kicking the player send him a info message.

### CODE REVIEW

File: src/Protocol/Protocol_1_12.cpp [hunk 1]
```diff
@@ -1526,9 +1546,9 @@ bool cProtocol_1_12::HandlePacket(cByteBuffer & a_ByteBuffer, UInt32 a_PacketTyp
 				case 0x14: HandlePacketBlockDig(a_ByteBuffer); return true;
 				case 0x15: HandlePacketEntityAction(a_ByteBuffer); return true;
 				case 0x16: HandlePacketSteerVehicle(a_ByteBuffer); return true;
-				case 0x17: break;  // Crafting Book Data - not yet implemented
+				case 0x17: HandlePacketCraftingBookData(a_ByteBuffer); return true;  // Crafting Book Data - not yet implemented
 				case 0x18: break;  // Resource pack status - not yet implemented
-				case 0x19: break;  // Advancement Tab - not yet implemented
+				case 0x19: HandlePacketAdvancementTab(a_ByteBuffer); return true;  // Advancement Tab - not yet implemented
```
Comment (madmaxoft) [inline src/Protocol/Protocol_1_12.cpp]: The comments should be removed from here

Comment (Seadragon91) [inline src/Protocol/Protocol_1_12.cpp]: Removed both comments.

File: src/Protocol/Protocol_1_12.cpp [hunk 2]
```diff
@@ -1296,6 +1296,26 @@ void cProtocol_1_12::SendUpdateBlockEntity(cBlockEntity & a_BlockEntity)
 
 
 
+void cProtocol_1_12::HandlePacketCraftingBookData(cByteBuffer & a_ByteBuffer)
+{
+	a_ByteBuffer.SkipRead(a_ByteBuffer.GetReadableSpace() - 1);
+	m_Client->GetPlayer()->SendMessageInfo("The feature crafing book (green) is not implemented.");
```
Comment (madmaxoft) [inline src/Protocol/Protocol_1_12.cpp]: I'm not sure about the English of this. I myself would have written `The green crafting book feature is not implemented yet.`

Need some native English speaker :)

Comment (Sdyessdev) [inline src/Protocol/Protocol_1_12.cpp]: Native English speaker here, I would write 

> The green crafting book feature is not implemented yet.

as well.

Comment (Seadragon91) [inline src/Protocol/Protocol_1_12.cpp]: Changed it, thanks :)

**Pull request merged**